docs/book/src/reference/versions.md

```diff
@@ -24,10 +24,10 @@ When a new Kubernetes minor release is available, we will try to support it in a
 the corresponding required changes in Cluster API are too invasive we won't backport the support and users have to wait 
 for the next Cluster API minor release.
 
-For example, Cluster API v1.5.0 would support the following Kubernetes versions:
+For example, Cluster API v1.6.0 would support the following Kubernetes versions:
```

Comment:
This is just an example we don't have to update it.

It will also be wrong as v1.6 will support 1.28

Reply:
undone